src/calculateRentalCost.js

```diff
@@ -28,7 +28,12 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
```

Comment:
`40 * days` transfer to separate variable. No need to calculate every time.

Reply:
ok! thx for advice!